backend/src/main/java/ai/verta/modeldb/experimentRun/subtypes/HandlerUtil.java

```diff
@@ -0,0 +1,59 @@
+package ai.verta.modeldb.experimentRun.subtypes;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicInteger;
+import org.apache.logging.log4j.Logger;
+import org.jdbi.v3.core.Handle;
+
+public abstract class HandlerUtil {
+
+  protected String buildInsertQuery(Map<String, Object> rmValueMap, String entityName) {
+    // Created comma separated field names from keys of above map
+    String[] fieldsArr = rmValueMap.keySet().toArray(new String[0]);
+    var commaFields = String.join(",", fieldsArr);
+
+    // Created comma separated query bind arguments for the values
+    // based on the
+    // keys of
+    // above the map
+    // Ex: VALUES (:project_id, :experiment_id, :name) etc.
+    var bindArguments =
+        String.join(",", Arrays.stream(fieldsArr).map(s -> ":" + s).toArray(String[]::new));
+
+    return String.format(
+        "insert into %s (%s) values (%s) ", entityName, commaFields, bindArguments);
```

Comment:
At first glance, this appears to be a SQL Injection vector in all three arguments passed here. Is there an input validation stage somewhere before this call? Should we consider moving validation to directly before constructing dynamic queries like this? Any method calling this method has the potential to introduce a vulnerability.

Reply:
Yes we have many other argument and logical validation before calling this. this is an internal parameters with validation so there is no any possibilities for SQL injection.